SPECS/geos.spec

```diff
@@ -11,7 +11,6 @@ Source1:        geos-check_doxygen_errors.cmake
 
 # Honor libsuffix
 Patch1:         geos-libsuffix.patch
-Patch2:         geos-3.9.1-heuristic-overlay.patch
```

Comment:
Justin and I found that Heuristic overlay was causing issues with Hootenanny.  He created this patch and then submitted the patch to their main repo.  Before we remove it, did you verify that his PR is in the 3.9.2 version?

Reply:
Yep, that's indeed why it had been removed:
https://github.com/libgeos/geos/blob/3.9.2/src/geom/HeuristicOverlay.cpp